cfn-resources/alert-configuration/Makefile

```diff
@@ -9,11 +9,11 @@ ldXflagsD=-s -w -X github.com/mongodb/mongodbatlas-cloudformation-resources/util
 
```

Comment:
not changed in this PR but in case you can do it here, all  ldXflagsD (debug) must not have  -s -w,   an example would be: 

ldXflagsD=-X github.com/mongodb/mongodbatlas-cloudformation-resources/util.defaultLogLevel=debug -X github.com/mongodb/mongodbatlas-cloudformation-resources/version.Version=${CFNREP_GIT_SHA}

Reply:
thanks for suggestion, adjusted.